From 10f42638ee83b91ae91508e35e34fba8c527d6c6 Mon Sep 17 00:00:00 2001
From: Hawking Zhang <Hawking.Zhang@amd.com>
Date: Thu, 25 May 2017 18:22:38 +0800
Subject: [PATCH 2048/2094] drm/amdgpu: correct the vram_base_offset on raven

vram_base_offset has already been initialized in vram_gtt_location

Change-Id: I01880b16db170090b242f815b93156ae4b35be92
Signed-off-by: Hawking Zhang <Hawking.Zhang@amd.com>
Reviewed-by: Roger.He <Hongbo.He@amd.com>
Acked-by: Huang Rui <ray.huang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
index 0496e08..9a03a0c 100644
--- a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
@@ -646,10 +646,6 @@ static int gmc_v9_0_sw_init(void *handle)
 		adev->vm_manager.num_level = 3;
 	amdgpu_vm_manager_init(adev);
 
-	/* base offset of vram pages */
-	/*XXX This value is not zero for APU*/
-	adev->vm_manager.vram_base_offset = 0;
-
 	return 0;
 }
 
-- 
2.7.4

